packages/viewer/src/components/viewer-box-query-tool/viewer-box-query-tool.tsx

```diff
@@ -252,7 +252,7 @@ export class ViewerBoxQueryTool {
   private setDefaultClearAndSelectOperation(): void {
     this.controller?.setOperationTransform((builder) => builder.select());
     this.controller?.setAdditionalTransforms([
-      (op) => op.where((q) => q.all()).deselect(),
+      (op) => op.where((q) => q.withSelected()).deselect(),
```

Comment:
We should figure out why this change is necessary.

Reply:
Agreed, I created https://vertexvis.atlassian.net/browse/PLAT-2506 to capture this issue